md/database-configuration.md

```diff
@@ -1,285 +1,114 @@
+- Starting with Bonita BPM 7.3.0, the same database is also used to store Bonita BPM Platform configuration information, in dedicated tables.
+- Business data are stored in a database connected to Bonita BPM. We recommend that you configure a different database scheme for this purpose.
 
-Bonita BPM Engine connection to the database is done through the Hibernate library (version 4.2).  
+Connections to the databases are done through the Hibernate library (version 4.2).  
 This provides a level of abstraction between the engine and the RDBMS.  
-Bonita BPM comes with a set of optimized initialization scripts for the [supported RDBMSs](https://customer.bonitasoft.com/support-policies).
-
-::: info
-Some configuration file detailed below are located under `setup` folder. See [Bonita BPM platform setup tool](BonitaBPM_platform_setup.md) for more information about platform configuration.
-:::
 
-::: info
-This page explains database configuration for Bonita BPM Engine data. If you are also using [business data objects](define-and-deploy-the-bdm.md), these can be stored in a separate database. By default, Bonita BPM comes with an h2 database. For production purposes, we recommend that you [configure a different database for business data](database-configuration-for-business-data.md).
-:::
+Bonita BPM comes with a set of optimized initialization scripts for the [supported RDBMSs](https://customer.bonitasoft.com/support-policies).
 
-#### Default h2 database
+#### Default H2 database
 
-Bonita BPM Studio, the WildFly bundle, the Tomcat bundle, and the deploy bundle come by default with an h2 RDBMS included in the package. The database is automatically created at first startup.
+Bonita BPM Studio, the WildFly bundle, the Tomcat bundle, and the deploy bundle come by default with an embedded H2 RDBMS. The database is automatically created at first startup, an can be used for all three purposes described earlier.
 
-h2 is only suitable for testing purposes.
-For production purposes, you must modify the configuration to use another RDBMS.
+However, H2 is only suitable for testing purposes. When building a production environment, you must switch to another RDBMS.  
 
 #### Steps to switch to another RDBMS
 
-In order to configure Bonita BPM to work with your RDBMS, you need to perform the following steps:
+As suggested earlier, if you use the BDM (Business Data Model) feature to store business data related to the scope of your applications and processes, you will need to configure two databases: one for Bonita BPM Platform configuration data and engine data, and one for business data.
 
-1. Create the database
-2. Specify the database vendor
-3. Add the JDBC driver
-4. Configure data source
-5. Remove h2
-6. Configure RDBMS-specific settings
+Here are the steps to follow:
+    1. Create the database
```

Comment:
bullet points do not display well: http://doc.rd.lan:50197/7.4?page=database-configuration